docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/vmware-workspace-one-source.md

```diff
@@ -0,0 +1,130 @@
+---
+id: vmware-workspace-one-source
+title: VMware Workspace One
+sidebar_label: VMware Workspace One
+tags:
+    - VMware Workspace One
+description: Collect the details of the devices and the corresponding applications of each device and send them to Sumo Logic.
+---
+import CodeBlock from '@theme/CodeBlock';
+import ExampleJSON from '/files/c2c/vmware-workspace-one/example.json';
+import MyComponentSource from '!!raw-loader!/files/c2c/vmware-workspace-one/example.json';
+import TerraformExample from '!!raw-loader!/files/c2c/vmware-workspace-one/example.tf';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/vmware_workspace_one.png')} alt="vmware-workspace-one-logo" width="60" />
+
+VMware is a leading software company specializing in virtualization and cloud technologies. It offers solutions that enhance IT infrastructure, streamline operations, and improve efficiency across various industries.
+
+VMware Workspace ONE is a digital workspace platform that combines unified endpoint management, access management, and application management. It allows organizations to securely deliver and manage apps on any device, simplifying device management, boosting security, streamlining user experience, and improving productivity.
+
+The VMware Workspace One source collects the details of the devices and the corresponding applications of each device and send them to Sumo Logic.
+
+## Data collected
+
+| Polling Interval | Data |
+| :--- | :--- |
+| 24 hours | [Search API](https://cn135.awmdm.com/API/help/Docs/Explore?urls.primaryName=MDM%20API%20V3#/DevicesV3/DevicesV3_SearchAsync) |
+| 24 hours | [App API](https://cn135.awmdm.com/API/help/Docs/Explore?urls.primaryName=MDM%20API%20V2#/AppsV2/AppsV2_GetAppListForDevicesAsync) |
+
+## Setup
+
+### Vendor configuration
+
+The VMware Workspace One source supports [O-Auth-based authentication](https://docs.omnissa.com/bundle/WorkspaceONE-UEM-Console-BasicsVSaaS/page/UsingUEMFunctionalityWithRESTAPI.html#create_an_oauth_client_to_use_for_api_commands_saas) and requires you to provide the **Endpoint URL**, **Auth URL**, **Client ID**, and **Client Secret** key to access the data.
+
+#### Endpoint URL
+
+The integration requires the configuration of the VMware Workspace One request endpoint URL, which will be the domain URL of the VMware Workspace console.
```

Comment:
```suggestion
Endpoint URL will be the domain URL of the VMware Workspace console. For example, `https://as135.awmdm.com`.
```

Reply:
First line of this statement is already covered Vendor configuration explanation para.